browse/templates/stats/monthly_downloads.html

```diff
@@ -0,0 +1,179 @@
+{%- extends "stats/base.html" -%}
+
+{% block title %}Monthly Download Rates{% endblock %}
+{% block header_h1 %}<h1><a href="{{ url_for('.home') }}">arXiv.org</a> &gt; Monthly Downloads</h1>{% endblock %}
+
+{% block content %}
+<h2>arXiv Monthly Download Rates [<a title="download monthly download CSV data" href="{{ url_for('browse.stats', command='get_monthly_downloads') }}">CSV</a>]</h2>
+
+<style>
+  svg {
+  font: 10px sans-serif;
+  }
+  path {
+  fill: steelblue;
+  }
+
+  .axis path,
+  .axis line {
+    fill: none;
+    stroke: lightgrey;
+    shape-rendering: auto;
+  }
+
+  .axis .minor line {
+    stroke: #777;
+    stroke-dasharray: 2,2;
+  }
+
+  .x.axis .minor {
+    stroke-opacity: .5;
+  }
+
+  .y.axis line {
+    stroke: lightgrey;
+    stroke-dasharray: 1,1;
+  }
+
+  .y.axis .minor {
+    stroke-opacity: .5;
+  }
+
+  .brush .extent {
+    stroke: #fff;
+    fill-opacity: .125;
+    shape-rendering: auto;
+  }
+</style>
+<script type="text/javascript">
+  var margin = {
```

Comment:
Similar to above, I wonder if a jinja {% include monthly_downloads.js %} would work?
That would afford a nicer IDE experience for editing the JS.

Reply:
I'm going to try this out.